spec/models/post_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { Post.new(title: 'This is my title', comments_counter: 2, likes_counter: 0) }
+
+  before { subject.save }
+
+  it 'The title should be present' do
+    subject.title.nil?
+    expect(subject).to_not be_valid
+  end
+
+  it 'The title should not exceed 250 characters' do
+    subject.title = 'This is my title' * 30
+    expect(subject).to_not be_valid
+  end
```

Comment:
* As mentioned by the previous CR it is a requirement to Add unit specs for all of your models' methods and validations.
* Currently you are missing tests for `like` and `comments` , Kindly add them 🙏

Reply:
***Hello!***
I have added tests for like and comments models as the previous reviewer suggested.